pkg/config/plugins/resources/postgres/config.go

```diff
@@ -43,6 +45,16 @@ type PostgresStoreConfig struct {
 	MaxReconnectInterval time.Duration `yaml:"maxReconnectInterval" envconfig:"kuma_store_postgres_max_reconnect_interval"`
 }
 
+func (cfg PostgresStoreConfig) ConnectionString() (string, error) {
+	mode, err := cfg.TLS.Mode.postgresMode()
+	if err != nil {
+		return "", err
+	}
+	escapedPass := strings.ReplaceAll(strings.ReplaceAll(cfg.Password, `\`, `\\`), `'`, `\'`)
+	return fmt.Sprintf(`host=%s port=%d user=%s password='%s' dbname=%s connect_timeout=%d sslmode=%s sslcert=%s sslkey=%s sslrootcert=%s`,
```

Comment:
Should we put all these values in single quotes?

Reply:
It's not clear to me whether any other params can have a ` `, `'` or `\` in them but yeah we can escape everything.

Reply:
Looks like at least DB name can have a space. Seems safe to quote them all. Thanks!